### ISSUE TITLE
Run script doesn't generate cucko_generator binary

### ISSUE THREAD

**Issue by danielrees18** [Brightify/Cuckoo#119]
When integrating Cuckoo using Cocoapods, the `run` script fails the Xcode build with the error 

```
> Referenced from: /Users/me/myproject/Pods/Cuckoo/Generator/cuckoo_generator
> Reason: image not found
```

The solution is to manually trigger the `build_generator` script to generate the `cuckoo_generator` binary.

```
$ cd Pods/Cuckoo/
$ ./build_generator
...
Compile Swift Module 'cuckoo_generator' (4 sources)
Linking ./.build/debug/cuckoo_generator
```

If you `ls -al Pods/Cuckoo/Generator/` you should now see the `cuckoo_generator` binary and Xcode will now succeed.

The expected behavior is that the `run` script does this automatically. 

Note: I only saw this behavior when integrating using Cocoapods and never had this problem using Carthage

**Comment by danielrees18** [comment]
After further digging, I've noticed the following behavior:

So the reason why the `run` script is failing is because it is checking for the `--clean` option or if the `cuckoo_generator` file does not exists.
```
if [[ "$1" = "--clean" || ! -e "$FILE_PATH" ]]; then
```

Well, in my case, I am not passing `--clean` and `cuckoo_generator`file exists at `Pods/Cuckoo/Generator/cuckoo_generator` so the script doesn't build the generator. This is the expected behavior.

 However, when I run `pod install`, this command doesn't actually remove the `cuckoo_generator` file, it is still there, but it does clean the `Generator/.build/` dir which is what Xcode is looking for.

This is the error that Xcode is returning
>`dyld: Library not loaded: /Users/drees/src/myProject/Pods/Cuckoo/Generator/.build/debug/libCYaml.dylib`

After manually building the `cuckoo_generator` the `.build` dir exists and Xcode runs just fine. 

The work around that I added for now is to modify the `run` script to check if the `.build` dir exists
```
BUILD_PATH="$SCRIPT_PATH/Generator/.build"
if [[ "$1" = "--clean" || ! -e "$FILE_PATH" || ! -d "$BUILD_PATH" ]]; then
```

The correct solution is to not move the `cuckoo_generator` from the `.build/` dir and leave the `run` script as is to reduce the number of variables being checked.
